lua/wire/stools/expression2.lua

```diff
@@ -818,144 +704,131 @@ elseif CLIENT then
-		lst:AddItem(selectnone)
-		function selectnone:DoClick()
-			selectedfiles = {}
+				for i = 1, #checkboxes do
+					if not checkboxes[i]:GetChecked() then checkboxes[i]:Toggle() end
+				end
+			end
+
+			height = height + selectall:GetTall() + 2
 
-			for i = 1, #checkboxes do
-				if checkboxes[i]:GetChecked() then checkboxes[i]:Toggle() end -- checkboxes[i]:SetChecked( false )
+			local selectnone = vgui.Create("DButton")
+			selectnone:SetText("Select none")
+			lst:AddItem(selectnone)
+			function selectnone:DoClick()
+				selectedfiles = {}
+				for i = 1, #checkboxes do
+					if checkboxes[i]:GetChecked() then checkboxes[i]:Toggle() end
+				end
 			end
-		end
 
-		height = height + selectnone:GetTall() + 2
+			height = height + selectnone:GetTall() + 2
 
-		local ok = vgui.Create("DButton")
-		ok:SetText("Ok")
-		ok:SetToolTip("Shortcut for this button: Right click anywhere")
-		lst:AddItem(ok)
-		function ok:DoClick()
-			local haschoice = false
-			for k, v in pairs(selectedfiles) do haschoice = true break end
-			if not haschoice then pnl:Close() return end
+			local ok = vgui.Create("DButton")
+			ok:SetText("Ok")
+			ok:SetTooltip("Shortcut for this button: Right click anywhere")
+			lst:AddItem(ok)
+			function ok:DoClick()
+				local haschoice = next(selectedfiles) ~= nil
+				if not haschoice then pnl:Close() return end
 
-			local datastr = WireLib.von.serialize(selectedfiles)
-			local numpackets = math.ceil(#datastr / 64000)
-			for i = 1, #datastr, 64000 do
+				local datastr = WireLib.von.serialize(selectedfiles)
 				net.Start("wire_expression2_download_wantedfiles")
-				net.WriteEntity(ent)
-				net.WriteBit(uploadandexit)
-				net.WriteUInt(numpackets, 16)
-				local data = datastr:sub(i, i + 63999)
-				net.WriteUInt(#data, 32)
-				net.WriteData(data, #data)
+					net.WriteEntity(ent)
+					net.WriteBit(uploadandexit)
+					net.WriteStream(datastr)
 				net.SendToServer()
-			end
 
-			pnl:Close()
-		end
+				pnl:Close()
+			end
 
-		height = height + ok:GetTall()
+			height = height + ok:GetTall()
 
-		local down = input.IsMouseDown(MOUSE_RIGHT)
-		function pnl:Think()
-			if not down and input.IsMouseDown(MOUSE_RIGHT) then
-				ok:DoClick()
+			local down = input.IsMouseDown(MOUSE_RIGHT)
+			function pnl:Think()
+				if not down and input.IsMouseDown(MOUSE_RIGHT) then
+					ok:DoClick()
+				end
+				down = input.IsMouseDown(MOUSE_RIGHT)
 			end
-			down = input.IsMouseDown(MOUSE_RIGHT)
-		end
 
-		pnl:SetTall(math.min(height + 2, ScrH() / 2))
-		lst:EnableVerticalScrollbar(true)
-		lst:StretchToParent(2, 23, 2, 2)
-		pnl:MakePopup()
-		pnl:SetVisible(true)
+			pnl:SetTall(math.min(height + 2, ScrH() / 2))
+			lst:EnableVerticalScrollbar(true)
+			lst:StretchToParent(2, 23, 2, 2)
+			pnl:MakePopup()
+			pnl:SetVisible(true)
+		end)
 	end)
 
 	--------------------------------------------------------------
 	function TOOL.BuildCPanel(panel)
-		local w, h = panel:GetSize()
+		local w = panel:GetSize()
```

Comment:
I don't like sacrificing code consistency and readability to make the linter happy.